assets/js/app/LocusZoom.js

```diff
@@ -151,5 +210,13 @@
         return response.promise;
     };
 
+    exports.isString = function(x) {
```

Comment:
I'm not suggesting creating a global `isString` function, I'm suggesting using the basic `typeof x === "string"` syntax. One could argue it's not quite as clean as wrapping in an `isString` method of some kind, but it's standard JS string type checking and therefore something that javascript developers should know and be familiar with.

If this general syntax is a big issue for you then by all means we can keep the method. At the very least it can be less verbose:

```javascript
exports.isString = function(x) {
  return (typeof x === "string");
}
```

Since any time you're evaluating a conditional and returning true or false you can just as easily return the evaluated conditional.

But in general I favor caution when wanting to define functions in our narrowly-scoped library purely for making javascript's general syntax prettier. Like all languages it has its warts and we may find ourselves on a slippery slope defining more syntax helpers if we're not careful.